class_main.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import json
+import os
+import shutil
+from typing import Tuple
+
+import aiofiles
+
+import GetterPriceBinance
+import mail_handler
+
+
+class main_app:
+    def __new__(cls):
+        """
+        Singleton. Sorry, sometimes I'm stupid.
+        """
+        if not hasattr(cls, "instance"):
+            cls.instance = super(main_app, cls).__new__(cls)
+        return cls.instance
+
+    def __init__(self):
+        self._mail_client = mail_handler.factory_mail_handler(mode="gmail")
+        self._binance_websocket = GetterPriceBinance.book_ticker_price_binance(
+            symbol="BTCUAH"
```

Comment:
better to introduce a constant in more appropriate place

Reply:
added in config file